### PR TITLE
add TheWulfPanda's teleport spell mods to masterlist

### PR DESCRIPTION
**Pull request by jb29**
Hi! Here's the second mod of the Teleport Spells: [Teleport - Faction Spells](https://www.nexusmods.com/skyrimspecialedition/mods/6991/)

I guess you could argue about the cleaning instructions because the last item in the list isn't a wild edit but a fix from [USSEP](https://www.nexusmods.com/skyrimspecialedition/mods/266). I thought there's no harm in adding it anyway.

The author added the flag `Don't Havok Settle` on the following references:
- `[REFR:02010521]`
- `[REFR:000CA72C]`
- `[REFR:000B3434]`

I don't understand why mists and walls would benefit from this flag, so I removed those references too.

**Edit:**
I added the last teleport mod, too: [Teleport - Towns and Villages](https://www.nexusmods.com/skyrimspecialedition/mods/7267/)

### CODE REVIEW

File: masterlist.yaml
Comment (MacSplody) [inline masterlist.yaml]: These load after rules are not needed as its only a weather region edit that is being overwritten, which isn't critical to `Ars Metallica.esp`

image

**Review by MacSplody — CHANGES_REQUESTED** [review]
Everything else is fine, just remove the load after rules for Ars Metallica.

File: masterlist.yaml
Comment (jb29) [reply masterlist.yaml]: Okay, I rebased my PR.

**Review by MacSplody — APPROVED** [review]

**Review by jb29 — COMMENTED** [review]
I'm not sure if assigning me means I should leave a review. So here it is. 😄 

The text is actually indented with 4 spaces now while everything else uses an indentation of 2 spaces.
But in case of multiline strings this is just cosmetical anyway. YAML ignores the indentation in multiline strings, so you could even write them without an indentation at all as long as the next key has the correct indentation again.
So technically, both versions are fine.

**Comment by MacSplody** [comment]
hmm... Are you not able to merge this? I must be remembering incorrectly, I could have sworn that I was able to merge after being approved.

**Comment by jb29** [comment]
No, I don't have the rights to do that.
I'm not even allowed to approve or request changes in a review, I can just comment them. 😄

**Comment by MacSplody** [comment]
Github is strange, why have approval if a team member has to merge it anyway.

**Comment by Ortham** [comment]
> Github is strange, why have approval if a team member has to merge it anyway.

It's probably because you can configure a repository to require reviews before a PR can be merged, though none of LOOT's repos are like that.

**Comment by jb29** [comment]
Some projects require multiple reviews, for example from a frontend and backend developer.

And in some commercial projects the PRs will be rolled out to a staging system after approval where they will be further tested by their stakeholders before they're finally allowed to be merged to the master.

**Pull request merged**
